src/Fantomas.Core.Tests/StructTests.fs

```diff
@@ -127,3 +127,24 @@ namespace meh
 
 type TupleStruct = (struct (string * string))
 """
+
+[<Test>]
+let ``struct empty type`` () =
+    formatSourceString
+        false
+        """
+type NameStruct =
+
+    struct
+
+    end
+"""
+        { config with MaxValueBindingWidth = 120 }
```

Comment:
For future reference: `MaxValueBindingWidth` has no influence on this test, it is preferred not to be included in the test.